src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix4x4.Impl.cs

```diff
@@ -232,7 +232,38 @@ public static Impl CreateBillboard(in Vector3 objectPosition, in Vector3 cameraP
 
                 result.X = axisX.AsVector4();
                 result.Y = axisY.AsVector4();
-                result.Z = axisZ.AsVector4();;
+                result.Z = axisZ.AsVector4();
+                result.W = Vector4.Create(objectPosition, 1);
+
+                return result;
+            }
+
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            public static Impl CreateBillboardLeftHanded(in Vector3 objectPosition, in Vector3 cameraPosition, in Vector3 cameraUpVector, in Vector3 cameraForwardVector)
+            {
+                // In a left-handed coordinate system, the object's positive z-axis is in the same direction as its forward vector,
+                // and spherical billboards by construction always face the camera.
```

Comment:
nit: It's beneficial to ensure corresponding comments are in the right-handed API where relevant, that way devs have context regardless of which they look at.